src/main/java/com/amuzil/omegasource/magus/skill/modifiers/data/MultiModifierData.java

```diff
@@ -0,0 +1,66 @@
+package com.amuzil.omegasource.magus.skill.modifiers.data;
+
+import com.amuzil.omegasource.magus.skill.modifiers.api.BaseModifierData;
+import com.amuzil.omegasource.magus.skill.modifiers.api.ModifierData;
+import net.minecraft.nbt.CompoundTag;
+import org.apache.logging.log4j.LogManager;
+
+public class MultiModifierData extends BaseModifierData {
+
+    private int count;
+
+    public MultiModifierData() {
```

Comment:
Be sure to document that the default is 1

Reply:
Ya, I'll get Aidan to help with his part of this